### DISCUSSION TITLE
Ignore ".git" hidden folder when uploading beatmap

### DISCUSSION THREAD

**Discussion by dudehacker** [ppy/osu#15372] [Ideas] [answered]
Problem statement:
osu doesn't have any version control, or support for team collaboration on a song compilation project.

Git can be used but the only issue is the '.git' folder also gets uploaded to osu server which isn't needed.

Solution:
Make osu ignore the ".git" folder when uploading map

I think this would be fairly simple to implement and would be very helpful feature for mappers.

**Comment by frenzibyte** [comment, marked as answer]
osu!(lazer) doesn't support uploading maps yet. If you meant the current client osu!(stable), report in https://github.com/ppy/osu-stable-issues/issues.

**Reply by peppy** [reply]
In addition, we plan to offer better version control as part of lazer.
